tests/rptest/tests/acls_test.py

```diff
@@ -173,6 +181,18 @@ def get_super_client(self):
                        sasl_mechanism=self.algorithm,
                        tls_cert=cert)
 
+    def check_permissions(self, pass_w_base_usr=False):
```

Comment:
Nit: I'd spell out the whole name: `pass_with_base_user`, but I don't feel strongly about it.

Reply:
Removed.